kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -365,6 +371,43 @@
       "Whether to enable REST Proxy V3 API. Default is true.";
   private static final boolean API_V3_ENABLE_DEFAULT = true;
 
+  public static final String RATE_LIMIT_ENABLE_CONFIG = "rate.limit.enable";
+  private static final String RATE_LIMIT_ENABLE_DOC =
+      "Whether to enable request rate-limiting. Default is false.";
```

Comment:
Why false?  Previously any rate limiting is on by default, I think we should make this on by default too as it's safer to have rate limiting in place, and then it can be disabled if needed.

Reply:
This configuration is false to make upgrading easier. People upgrading to a newer version won't have to do additional work. They then can enable it at their own convenience.